### PR TITLE
Use case wrapper facelift INTER-148 

### PR DESCRIPTION
**Pull request by JuroUhlar**
* Implemented use case wrapper design
* Small nav visual fixes
* Don't show the Restart button on the homepage and playground
* Learn more cards wrap, for now, will come back to horizontal scrolling later after discussing with Design
* ran `prettier:fix` on the whole `src` folder
* Put all copy together into `content.tsx` file

Figma: https://www.figma.com/file/Rw0BBJ46Fpbx5s5lWflhVy/Fingerprint-Hub-Demo-Project?node-id=428%3A64127&mode=dev
Demo: https://staging.fingerprinthub.com/

### CODE REVIEW

File: src/client/components/common/UseCaseWrapper/UseCaseWrapper.tsx [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+  articleURL?: string;
+  listItems?: readonly React.ReactNode[];
+  children: React.ReactNode;
+  hideSrcListItem?: boolean;
+  hideDivider?: boolean;
+  mentionResetButton?: boolean;
+  returnUrl?: string;
+  contentSx?: React.CSSProperties;
+  moreResources?: UseCase['moreResources'];
+};
```
Comment (TheUnderScorer) [inline src/client/components/common/UseCaseWrapper/UseCaseWrapper.tsx:23]: Question - wouldn't it be more efficient just pass `UseCase` to the props?

**Review by TheUnderScorer — APPROVED** [review]
I love the new UI, nice work @JuroUhlar!

File: src/client/components/common/UseCaseWrapper/UseCaseWrapper.tsx (hunk 1 above)
Comment (JuroUhlar) [reply src/client/components/common/UseCaseWrapper/UseCaseWrapper.tsx:23]: Yeah, I was thinking about it, but it's also kind of nice to have the flexibility to assemble it Lego-style. I can make Playground using UseCaseWrapper, without modeling it as a use case (which it isn't). But if you prefer I can do that, or simplify it later.

**Review by ilfa — DISMISSED** [review]

**Review by makma — APPROVED** [review]

**Comment by JuroUhlar** [comment]
> I love the new UI, nice work @JuroUhlar!

Thanks, Sabrina drew it! :)

**Review by TheUnderScorer — APPROVED** [review]

**Pull request merged**